warehouse/redshift.go

```diff
@@ -35,6 +35,7 @@ var (
 )
 
 func NewRedshift(c *config.Config) *Redshift {
+	log.Printf("Config flag S3Only is on, data will not be loaded to Redshift")
```

Comment:
@jameremo I don't understand how the config here has `S3Only` set to `true` always?

Reply:
It doesn't, this is a completely embarrassing bit hurried coding without testing.  😬 Thank you for catching it.